tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"lib/helper",
+	"globalize/date"
+], function( $, Globalize, helper ) {
+
+return $.extend( helper, {
+	getAttributes: function( locale ) {
```

Comment:
`ui/widgets/calendar.js` does provide the `parse` property, so this seems just missing in this helper (along with missing tests).